docs/upgrades/unreleased.md

```diff
@@ -113,3 +113,24 @@
 * Introduce a new _lib/graphql-editing-service.ts_ file to initialize a _graphQLEditingService_ to support a new Editing Metadata Mode. Can be done by adding this file from the latest version introduced in _nextjs-xmcloud_ base template.
 
 * Update _lib/page-props-factory/plugins/preview-mode_ plugin to support a new Editing Metadata Mode. Can be done by replacing this file with the latest version introduced in _nextjs-xmcloud_ base template.
+
+* To support editing for fields in Pages, the new editing metadata architecture relies on the new metadata property 'field.metadata' (instead of on 'field.editable', which won't be used in this scenario). If you are using the new editing arhitecture in Pages (EditMode.Metadata) and have custom field component that manipulates or relies on 'field.editable' in some way, it may need to be reworked. Experience Editor still relies on 'field.editable', so it needs to be supported. See example below from SXA's Banner component:
+
+    ```ts
+        ...
+        export const Banner = (props: ImageProps): JSX.Element => {
+            ...
+              const isMetadataMode = sitecoreContext?.editMode === EditMode.Metadata;
```

Comment:
let's show how to access _sitecoreContext_ variable, and add required imports for _useSitecoreContext_ and _EditMode_  variables